Signal/src/AppDelegate.m

```diff
@@ -551,12 +551,12 @@ - (void)application:(UIApplication *)application
                                                      handler:^(UIAlertAction *_Nonnull action){
 
                                                      }]];
```

Comment:
Is this if you use the force-touch compose shortcut from the app icon on an unregistered device?

Reply:
think so.